metrics/base-metrics/src/test/java/org/eclipse/microprofile12/metrics/base/BaseMetricsTest.java

```diff
@@ -55,10 +55,14 @@ public void testBasePrometheusMetrics() throws IOException {
                 .request(TEXT_PLAIN)
                 .get();
         String responseText = response.readEntity(String.class);
-        assertTrue(responseText.contains("# TYPE base:thread_max_count"));
-        assertTrue(responseText.contains("# TYPE base:thread_count"));
-        assertTrue(responseText.contains("# TYPE base:cpu_system_load_average"));
-        assertTrue(responseText.contains("# TYPE base:jvm_uptime"));
+        assertTrue(responseText.contains("# TYPE base:thread_max_count") 
+                || responseText.contains("# TYPE base_thread_max_count"));
+        assertTrue(responseText.contains("# TYPE base:thread_count") 
+                || responseText.contains("# TYPE base_thread_count"));
+        assertTrue(responseText.contains("# TYPE base:cpu_system_load_average") 
+                || responseText.contains("# TYPE base_cpu_systemLoadAverage"));
```

Comment:
I was myself wondering if this is correct, so I took a look into the MP Metrics 2.0 spec. It says:

> Metric names with camelCase are no longer converted to snake_case for OpenMetrics output.

So yes, this is correct.